### DISCUSSION TITLE
Icon's Color Poll

### DISCUSSION THREAD

**Discussion by daiyam** [VSCodium/vscodium#1430] [Polls]
Vote for VSCodium's next icon!

Here, you can vote for the icon's color.
The Insiders color will be the 2nd most voted color (or 3rd if the colors are to close to each others).

**Don't forget to vote for the shape at https://github.com/VSCodium/vscodium/discussions/1429.**

The vote ends the 1st April at midnight.

The final icon will the combination of the winners.

|        |                                                           codium                                                            |                                                              c                                                              |                                                           derive                                                            |                                                         simplified                                                          |                                                            style                                                            |
| ------ |:---------------------------------------------------------------------------------------------------------------------------:|:---------------------------------------------------------------------------------------------------------------------------:|:---------------------------------------------------------------------------------------------------------------------------:|:---------------------------------------------------------------------------------------------------------------------------:|:---------------------------------------------------------------------------------------------------------------------------:|
|        |                                                                                                                             |                                                                                                                             |                                                                                                                             |                                                                                                                             |                                                                                                                             |
| blue   | <img src="https://user-images.githubusercontent.com/587742/222109292-66ca04c1-b5f2-467a-9b23-eed23bda5027.png" width="128"> | <img src="https://user-images.githubusercontent.com/587742/222109252-c693837c-3743-4217-8ffe-7240a08b6455.png" width="128"> | <img src="https://user-images.githubusercontent.com/587742/222109310-99b1bd35-b7d5-4030-8335-66c523ab3b0f.png" width="128"> | <img src="https://user-images.githubusercontent.com/587742/222109325-0ac03458-1e17-4b5d-83a4-fe6e637fd89c.png" width="128"> | <img src="https://user-images.githubusercontent.com/587742/222109343-ac773ce3-cdf7-43a1-89d9-68090ecccef5.png" width="128"> |
|        |                                                                                                                             |                                                                                                                             |                                                                                                                             |                                                                                                                             |                                                                                                                             |
| green  | <img src="https://user-images.githubusercontent.com/587742/222109297-7fc9682e-e1ae-4509-953c-2df39337fe10.png" width="128"> | <img src="https://user-images.githubusercontent.com/587742/222109273-1234290e-ba94-41bd-be1b-4591e129d3eb.png" width="128"> | <img src="https://user-images.githubusercontent.com/587742/222109316-2c381c96-20e6-407b-a133-d74e2ed1523b.png" width="128"> | <img src="https://user-images.githubusercontent.com/587742/222109330-cd527516-8223-4da1-a3ed-fb87aba39c39.png" width="128"> | <img src="https://user-images.githubusercontent.com/587742/222109348-0acd40b6-2193-44db-8029-bc8f008b3c50.png" width="128"> |
|        |                                                                                                                             |                                                                                                                             |                                                                                                                             |                                                                                                                             |                                                                                                                             |
| yellow | <img src="https://user-images.githubusercontent.com/587742/222109306-a657bea5-b7a8-4893-8513-b923024d7a43.png" width="128"> | <img src="https://user-images.githubusercontent.com/587742/222109289-7dc7f9f7-60f2-4315-9867-758d9cc89ebb.png" width="128"> | <img src="https://user-images.githubusercontent.com/587742/222109322-c894be1e-7484-443c-b308-d2a73d7e19e3.png" width="128"> | <img src="https://user-images.githubusercontent.com/587742/222109339-6a9cee57-5f54-411e-9228-383a67ba3d7a.png" width="128"> | <img src="https://user-images.githubusercontent.com/587742/222109359-7f1d5547-9299-4c22-a7cc-0934878d00f5.png" width="128"> |
|        |                                                                                                                             |                                                                                                                             |                                                                                                                             |                                                                                                                             |                                                                                                                             |
| orange | <img src="https://user-images.githubusercontent.com/587742/222109298-f0c8b58a-e383-41d3-a821-b05790ba4d68.png" width="128"> | <img src="https://user-images.githubusercontent.com/587742/222109275-f7b1321e-bcb6-4de8-b5f6-2b2e144c2908.png" width="128"> | <img src="https://user-images.githubusercontent.com/587742/222109317-a0c5da53-224d-4813-85d2-6c99f2ad86e7.png" width="128"> | <img src="https://user-images.githubusercontent.com/587742/222109333-6df8a2b7-1d33-4ed7-b264-82c980e857f3.png" width="128"> | <img src="https://user-images.githubusercontent.com/587742/222109350-a381ccd8-0f87-4317-95cf-436afed4d6a9.png" width="128"> |
|        |                                                                                                                             |                                                                                                                             |                                                                                                                             |                                                                                                                             |                                                                                                                             |
| red    | <img src="https://user-images.githubusercontent.com/587742/222109303-e263e5c4-79bc-41e8-a60e-10077022fef1.png" width="128"> | <img src="https://user-images.githubusercontent.com/587742/222109285-13541083-b9a2-4802-9593-53fa76a41093.png" width="128"> | <img src="https://user-images.githubusercontent.com/587742/222109320-d8a892aa-4f7c-4511-93e4-72d32df9b539.png" width="128"> | <img src="https://user-images.githubusercontent.com/587742/222109337-845011c1-915b-4ee1-ba4d-c5d6a2c98cec.png" width="128"> | <img src="https://user-images.githubusercontent.com/587742/222109356-4d99be8b-e7f2-4bfc-bcd7-9d426b010a86.png" width="128"> |
|        |                                                                                                                             |                                                                                                                             |                                                                                                                             |                                                                                                                             |                                                                                                                             |
| purple | <img src="https://user-images.githubusercontent.com/587742/222109299-1334dc4e-0a11-4561-88c1-6b03edd33be3.png" width="128"> | <img src="https://user-images.githubusercontent.com/587742/222109280-6c25e655-5661-47ea-baaf-8c092a8ea667.png" width="128"> | <img src="https://user-images.githubusercontent.com/587742/222109319-4888512e-a81d-4c25-a652-1bc507e81aee.png" width="128"> | <img src="https://user-images.githubusercontent.com/587742/222109334-47d43ee7-76c6-468e-91f6-cf3a2b8640ac.png" width="128"> | <img src="https://user-images.githubusercontent.com/587742/222109352-2e771538-a0f9-4059-aedd-53eedc577987.png" width="128"> |
|        |                                                                                                                             |                                                                                                                             |                                                                                                                             |                                                                                                                             |                                                                                                                             |
| brown  | <img src="https://user-images.githubusercontent.com/587742/222109294-828fa37b-e189-4cfb-a4a6-f64ee66bb251.png" width="128"> | <img src="https://user-images.githubusercontent.com/587742/222109270-10b682d2-6010-45c3-ae31-e04f53d141bc.png" width="128"> | <img src="https://user-images.githubusercontent.com/587742/222109313-2bc220e6-91d2-4201-bd59-b7283ef227ae.png" width="128"> | <img src="https://user-images.githubusercontent.com/587742/222109327-e305d37c-268b-4a5f-a0de-98cb77a0d334.png" width="128"> | <img src="https://user-images.githubusercontent.com/587742/222109346-3d371d70-46d1-4b00-bf56-f77ff2463652.png" width="128"> |

**Don't forget to vote for the shape at https://github.com/VSCodium/vscodium/discussions/1429.**

**Comment by AuraHerreroRuiz** [comment]
Just a thought, maybe it would be nice to take color blindness into account when choosing the insiders icon color? I'm not color blind, but from what I've been able to read, Red and Green, or Blue and Yellow icons could cause problems.

**Reply by daiyam** [reply]
Some combinations could be tricky like blue-purple and red-brown. Not sure about others.

**Comment by Derkades** [comment]
Most of the icons on my taskbar are already blue and green... I like the warm orange color!

**Reply by WofWca** [reply]
Idk, it doesn't associate with code IMO.

**Comment by thatoddshade** [comment]
Nobody voted for brown.

**Reply by ThatOneCalculator** [reply]
poopy ide :exploding_head:

**Reply by DanielProg39** [reply]
It would be grater if brown theme would be more yellow, this will make it look like bronze

**Comment by DanielProg39** [comment]
Ah, I like both blue and green, what to choose? 🤔

**Reply by lilly-lizard** [reply]
imo anything but blue because so many apps have blue logos already! I like blue but I don't want everything to be blue haha

**Reply by DanielProg39** [reply]
> imo anything but blue because so many apps have blue logos already! I like blue but I don't want everything to be blue haha

Yup, I chose blue btw

**Reply by lilly-lizard** [reply]
> Yup, I chose blue btw

haha fair enough. looks like its winning by a lot too!

**Comment by clay-rip** [comment]
Red frfr looks good, but I feel like orange could be a bit more vibrant

**Comment by DanielProg39** [comment]
I think it would be better if there was a setting to choose an accent colour, so that it changes some in-app highlighting and an icon colour.

**Reply by clay-rip** [reply]
This.

**Comment by daiyam** [comment]
Preview is available, see comment at https://github.com/VSCodium/vscodium/discussions/1218#discussioncomment-5357728

**Comment by kxrn0** [comment]
vote green, you fools

**Comment by unixtensor** [comment]
Yellow, orange, brown, or red. These stand out from VSCode's blue, Visual Studio's purple, and VSCode's Insiders edition green.

**Comment by dpst999** [comment]
Make it **red**, so it will be **team blue** (VSCode) vs **team red** (VSCodium)

**Reply by DanielProg39** [reply]
Lol, makes sense)

**Comment by daiyam** [comment]
I will close the votes 12 hours earlier since the next version is already out (a week early). So you have until 12:00 PM UTC to cast your votes.

**Comment by dpst999** [comment]
Can we have all colors & icons ? So that we can change it upto our preference

**Reply by daiyam** [reply]
It's more an how than anything else. Here a discussion about that https://github.com/VSCodium/vscodium/discussions/1218#discussioncomment-5190293

**Comment by daiyam** [comment]
Thank you to every voters to have participated in this vote. :clap:
